### DISCUSSION TITLE
sm predictions from flavio for CMS B0→K*mumu angular analysis

### DISCUSSION THREAD

**Discussion by pdinie831** [flav-io/flavio#287] [Q&A]
Dear Flavio authors,

 I'm writing on behalf of the CMS group performing the angular analysis of the B0->K*mumu decay.
As the analysis is approaching its final stages, we would like to compare our results to the SM predictions obtained from Flavio for the FL and P_i parameters.
Following the instructions on the webpage we made a first attempt, which you can find here [1].
We would like to know if we are using flavio (version 2.5.5) correctly, and the meaning of this warning that we get while we execute our
flavio script: "UserWarning: The QCDF corrections should not be trusted for q2 above 6 GeV^2".

We thank you in advance for your help and suggestions.
If it's more convenient, we are of course available for discussion.

Thanks again,
Paolo Dini

[1]
https://github.com/pdinie831/PredictionFlavio/tree/main
[<https://github.com/pdinie831/PredictionFlavio/tree/main>](https://github.com/pdinie831/PredictionFlavio/tree/main)

**Comment by DavidMStraub** [comment]
Hi Paolo. I'll leave it to @peterstangl to address your question, but would just like to point out the discussions in https://github.com/flav-io/flavio/issues/198 and https://github.com/flav-io/flavio/issues/229 that might be related, even if it's a different decay.

**Comment by peterstangl** [comment]
Hi Paolo,

in flavio, parts of the non-local b->sll matrix elements are implemented following QCD factorization (QCDF), which can only be used for $q^2$ up to ~ 6 or7 $\text{GeV}^2$ (see  [arXiv:hep-ph/0008255](https://arxiv.org/abs/hep-ph/0008255), [arXiv:hep-ph/0106067](https://arxiv.org/abs/hep-ph/0106067) [arXiv:hep-ph/0412400](https://arxiv.org/abs/hep-ph/0412400)). So for the bin [6,8] $\text{GeV}^2$, the theory prediction from flavio is not reliable and should not be used. This is the meaning of the warning "UserWarning: The QCDF corrections should not be trusted for q2 above 6 GeV^2".

You should also not use the flavio prediction for the region between the two narrow charmonium resonances (your bin [10.09, 12.86] $\text{GeV}^2$).

Above the narrow charmonium resonances, the theory predictions are only valid for sufficiently global, i.e. integrated, observables in the region above ~ 15 $\text{GeV}^2$, see [arXiv:1101.5118](https://arxiv.org/abs/1101.5118). In practice this means that comparisons between theory and experiments in the high $q^2$ region should use the widest possible bins and these bins should start not considerably below 15 $\text{GeV}^2$. This means that also for your bin [14.18, 16] $\text{GeV}^2$, the theory prediction is not reliable. For comparison, LHCb uses the bin [15, 19] $\text{GeV}^2$ in their analyses of the B0->K*mumu decay in the high $q^2$ region.

Let me also mention that the bin [1,2] $\text{GeV}^2$ is not ideal due to the presence of the $\phi(1020)$ at 1 $\text{GeV}^2$, which is not included in the theory prediction. For a comparison with theory it would be better to start the bin at 1.1 $\text{GeV}^2$ (which is also what LHCb is doing).

Finally just a small comment about your code: the flavio function `sm_uncertainty` has an optional argument `N` to specify the number of random evaluations of the observable when determining the uncertainty. The default value is `N=100` and the relative accuracy of the uncertainty is given by $1/\sqrt{2N}$. So to get more accurate results for the uncertainties, you should use a larger `N` (e.g 5000 for a 1% relative accuracy). (see the API documentation https://flav-io.github.io/apidoc/flavio/functions.m.html#flavio.functions.sm_uncertainty)
